controllers/utils/conditions.go

```diff
@@ -11,13 +11,17 @@ type ConditionType string
 
 // ConditionTypes define the different types of conditions that will be set
 var ConditionTypes = struct {
+	// CGU
 	BackupSuceeded     ConditionType
 	ClustersSelected   ConditionType
 	PrecacheSpecValid  ConditionType
 	PrecachingSuceeded ConditionType
 	Progressing        ConditionType
 	Succeeded          ConditionType
 	Validated          ConditionType
+
+	// IBGU
+	ManifestsCreated ConditionType
```

Comment:
Does it make sense to have a totally seperate ConditionTypes struct e.g `ConditionTypesIBGU` for ibgu? That way we wont accidentally pull in "invalid" types and just reads clearer?

Reply:
Right, we certainly don't want to copy functions just for this.

Reply:
I will think about some kind of interface for a future PR

Reply:
next PR sounds good to me no problem